src/commands/schema/push.js

```diff
@@ -0,0 +1,127 @@
+const FaunaCommand = require("../../lib/fauna-command.js");
+const fetch = require("node-fetch");
+const fs = require("fs");
+const path = require("path");
+const { Flags, ux } = require("@oclif/core");
+const FormData = require("form-data");
+
+const FILE_LIMIT = 256;
+const FILESIZE_LIMIT_BYTES = 32 * 1024 * 1024;
+
+class PushSchemaCommand extends FaunaCommand {
+  async run() {
+    const {
+      connectionOptions: { domain, port, scheme, secret },
+    } = await this.getClient();
+    const dir = this.flags.dir;
+    const filenames = fs.readdirSync(dir);
+    var totalsize = 0;
+    const files = filenames
+      .filter(
+        (filename) =>
+          filename.endsWith(".fsl") &&
+          !fs.statSync(path.join(dir, filename)).isDirectory()
```

Comment:
files can be many things. this should probably be `isFile`, so that symlinks and other weird things aren't uploaded.
```suggestion
          fs.statSync(path.join(dir, filename)).isFile()
```

Reply:
I think if we can open it and suck bytes out of it like a file we can upload it. I wouldn't use symlinks myself but I would expect it to read file contents through a symlink.

Plus, if you try to handle every goddamn unix this-or-that then your head will explode. Well-established fact. The way to deal with people doing weird dumb stuff is to tell them not to do that.